### ISSUE TITLE
[youtube] "Last message repeated XXX times" (or more)

### ISSUE THREAD

**Issue by marabu88** [rg3/youtube-dl#16513]
### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.05.18*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2018.05.18**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [x] Question
- [ ] Other

**I can not bring the log, since this does not happen with all streams! besides, these are living streams and I can not reproduce this problem! in addition, some streams are very time-consuming. At the end only this line is "Last message repeated XXX times". Nothing special except this.**

Some live streams do not end correctly. I get the message "Last message repeated N times". "N" can be really big (N > 1000) and it really downloaded each time last message from Youtube. Such hanging streams heavily load the network when there are a lot of them.

For download streams from Youtube i use last version yutube-dl and use command string like this:
`youtube-dl.exe --hls-use-mpegts https://www.youtube.com/watch?v=XXXXXXXXXXX`

Why some streams not ended correctly and how fix it? I can set some limit for this?

I REALLY tried to find the answer to my question and did not find it! If you think that this problem is described and solved - then PLEASE post here a link to this solution. It is very difficult for me to write in English, which does not detract from the importance of my communication.

P.s. I have a suspicion that this happens to streams that have been deleted

**Comment by marabu88** [comment]
Only i have this problem? No any ideas how fix it?

**Comment by dstftw** [comment]
There is not enough context for any reasoning. Anyway `Last message repeated` comes from ffmpeg so you may ask there. Closing for now.

**Comment by marabu88** [comment]
I think this is a problem with YouTube, not ffmpeg or youtube-dl. but you can somehow automatically stop recording after a certain number of repetitions ???
if you want to reproduce this problem - just record any random streams of users. It often happens that the broadcast is complete, but YouTube continues to play a static picture. I think that this is what leads to this behavior.

**Comment by ZuyingWo** [comment]
here is what I did. Hoping it can be your reference.
===========
In my local environment, this happens when FFMPEG is called to merge audio and video files.

when I download youtube via urls, most audio tracks are saved as "webm" file, and most video tracks are "mp4" file.

And this ERROR always occurs when ffmpeg merging "webm" and "mp4".

I don't know why, but audio and video are merged successfully when I convert  "webm" file to "m4a" file at first, and then merge "m4a" and "mp4".
Here are the commands I used.

> ffmpeg -i xxxxxx.webm -vn xxxxxx.m4a
> ffmpeg -i xxxxxx.mp4 -i xxxxxxx.m4a -c copy xxxxxxx.mov
